pkg/asset/manifests/azure/cluster.go

```diff
@@ -93,25 +66,50 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 		nodeSubnetID = "UNKNOWN"
 	}
 
+	//apiServerLBFrontendConfigName := azure.InternalFrontendIPv4ConfigName
+	apiServerLBFrontendConfigName := fmt.Sprintf("%s-internal-frontEnd", clusterID.InfraID)
+	//apiServerBackendPoolName := azure.InternalBackendAddressPoolIPv4Name
+	apiServerBackendPoolName := fmt.Sprintf("%s-internal", clusterID.InfraID)
+	if installConfig.Config.Networking.IsIPv6() && !installConfig.Config.Networking.IsDualStack() {
```

Comment:
```
if installConfig.Config.Networking.IsIPv6() && installConfig.Config.Networking.IsDualStack() {
```
We will be adding support for dualstack first, so this can be changed to: